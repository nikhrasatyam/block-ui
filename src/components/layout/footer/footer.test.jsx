import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'mobx-react';


import Footer from './footer';
import ConfigStore from '../../../stores/config.store';


describe('Footer Test Cases', () => {
  let configStore;
  beforeEach(async () => {
    configStore = new ConfigStore();
    configStore.setI18n(text);
  });
  afterEach(() => { configStore = null; });

  test('should match snapshot', () => {
    const tree = renderer.create(<Provider configStore={configStore}><Footer /></Provider>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
