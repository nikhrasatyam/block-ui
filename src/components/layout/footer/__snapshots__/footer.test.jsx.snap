// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Test Cases should match snapshot 1`] = `
<div
  className="sc-ifAKCX cFlEyZ"
>
  <div
    className="sc-EHOje dODvMY"
  >
    <span
      className="sc-bZQynM goJlMT"
    >
      If you're experiencing technical issues, please
       
      <a
        className="sc-bdVaJa eHRVLd"
        onClick={[Function]}
      >
        Report Issue
      </a>
    </span>
    <span
      className="sc-bZQynM goJlMT"
    >
      This site is protected by reCAPTCHA and the Google
    </span>
    <span
      className="sc-bZQynM goJlMT"
    >
      <a
        href="https://policies.google.com/privacy"
      >
        Privacy Policy
      </a>
       
      and
       
      <a
        href="https://policies.google.com/terms"
      >
        Terms of Service
      </a>
       
      apply
      .
    </span>
  </div>
</div>
`;
